docs/triggers/link-triggers.md

```diff
@@ -0,0 +1,120 @@
+## Link Triggers
+
+A Link Trigger is an interactive Trigger that activates when a Shortcut is clicked in the Slack client. When a Shortcut Trigger is created, its API response returns a `shortcut_url` which can be used in Slack to show a button. Clicking on this button will activate the associated Workflow. A Link Trigger
```

Comment:
Looks like we need to do a sweep on this doc to make sure we're consistently referring to this as a Link Trigger and not a Shortcut Trigger. I think the only cases where we should use the term `shortcut` atm is when talking about the configuration object or the `shortcut_url`